app/models/content_item.rb

```diff
@@ -10,7 +10,7 @@ def initialize(content_item_data)
     @content_item_data = content_item_data
   end
 
-  %i[base_path title description content_id].each do |field|
+  %i[base_path title description content_id schema_name].each do |field|
```

Comment:
We should probably be using `document_type` for this. Schema describes the structure, not the content.